tests/integration/test_render.py

```diff
@@ -105,21 +104,36 @@ def test_items(self):
     def test_items_custom_colors(self, item_background_color, item_text_color):
         self.load_scenario(item_background_color, item_text_color)
 
-        items = self._get_items()
-
-        self.assertEqual(len(items), 3)
-
         color_settings = {}
         if item_background_color:
             color_settings['background-color'] = item_background_color
         if item_text_color:
             color_settings['color'] = item_text_color
 
+        self._test_items(color_settings=color_settings)
+
+    def _test_items(self, color_settings={}):  # pylint: disable=dangerous-default-value
```

Comment:
Please use `color_settings=None` as the default (together with `color_settings = color_settings or {}` in the function), rather than suppressing the warning.